File: 1-two-sum/two-sum.php
class Solution {

    /**
     * @param Integer[] $nums
     * @param Integer $target
     * @return Integer[]
     */
    function twoSum($nums, $target) {
        $result = array(0, 0);

    $length = count($nums);

    for ($i = 0; $i < $length; $i++) {
        for ($j = $i + 1; $j < $length; $j++) {
            if ($nums[$i] + $nums[$j] == $target) {
                $result[0] = $i;
                $result[1] = $j;
                
            }
        }
    }

    // Return an array with -1 if no solution is found
    return $result;

    }
}

File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
     HashMap <Integer,Integer> map = new HashMap<>();
      
        for(int i=0;i<nums.length;i++){
       
           int comp = target- nums[i];

           if(map.containsKey(comp)){
               
               return new int[]{map.get(comp),i};

           }

           map.put(nums[i],i);        

        }
        
       return new int[]{-1,-1};  // NO target found in the array 

    }
}